### DISCUSSION TITLE
Multiple Triggers

### DISCUSSION THREAD

**Discussion by Veteran66** [Key2Joy/Key2Joy#71] [General]
Hi Luttje

i test the Multiple Triggers, the Keyboard Keys /Ctrl+F1) don`t release after Push/Release the Game Controller Button (A).
I test the Version 6 and / Alpha
![Key2Joy1](https://github.com/luttje/Key2Joy/assets/58970241/ed78098b-b6d3-4201-aa78-3aee96bcb057)

**Comment by luttje** [comment]
Thanks for helping out by reporting this! You're absolutely correct. 

I've created an issue in #74 for this bug.

**Reply by Veteran66** [reply]
Hi luttje, any News for this bug :)
Can i wride the Multible Push/Release in the profile.k2j.json script?
When yes you have a Example?

**Reply by luttje** [reply]
No news and I'm not sure if there's a way to work around the problem
